new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -93,6 +93,23 @@
   public static final String VOYAGER_CHART_NAME = "voyager";
   public static final String VOYAGER_CHART_VERSION = "12.0.0";
 
+  // ELK Stack and WebLogic logging exporter constants
+  public static final String ELASTICSEARCH_NAME = "elasticsearch";
+  public static final String ELASTICSEARCH_IMAGE = ELASTICSEARCH_NAME + ":6.8.0";
```

Comment:
seems like an old version of ES to test?

Reply:
let me try the latest version

Reply:
Hi Mark, I have upgraded the ELK Stack(Elasticsearch, Kibana and Logstash) version to 7.8.1, latest version